src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+h1,
```

Comment:
https://www.w3schools.com/cssref/css_default_values.asp

Reply:
don't add padding: 0 to all tags!

Reply:
fixed